app/views/RoomInfoView/index.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { View, Text, ScrollView } from 'react-native';
+import {
```

Comment:
Unnecessary changes.

Reply:
Sorry, I didn't remove the temprory changes. Now I have fixed them